src/components/CalendarBody.tsx

```diff
@@ -100,6 +103,9 @@ function _CalendarBody<T extends ICalendarEventBase>({
 }: CalendarBodyProps<T>) {
   const scrollView = React.useRef<ScrollView>(null)
   const { now } = useNow(!hideNowIndicator)
+  const hours = minHour < maxHour && minHour >= 0 && maxHour <= 23
+      ? Array.from({ length: maxHour - minHour + 1 }, (_, i) => minHour + i)
+      : Array.from(Array(24).keys())
```

Comment:
Can you validate it in `CalendarContainer.tsx`? like

```ts
if (minHour >= maxHour) { 
  throw new Error('minHour should be greater than maxHour')
}
if (minHour <= 0) { 
  // other error message
}
// other checks
```

I think it's more intuitive to provide developers comprehensive error message than defaulting the basic.